create/how_to/updating.md

```diff
@@ -0,0 +1,314 @@
+# Update to the Latest MeiliSearch Version
+
+Currently, MeiliSearch databases can only be opened by the MeiliSearch version you used to create them. The following guide will walk you through all the steps to **migrate an existing database** from an older version of MeiliSearch to the most recent one.
+
+If you already have a MeiliSearch database with some data you don’t want to lose, you are in the right place!
+
+## Verify your database version
+
+Before we begin, you need to **verify the version of MeiliSearch that's compatible with your database**, i.e. the version that indexed the data. You can do so by launching a MeiliSearch instance:
+
+```bash
+./meilisearch
+```
+
+If MeiliSearch launches successfully, use the [get version endpoint](/reference/api/version.md), note your `pkgVersion`, and [proceed to the next step](#proceed-according-to-your-database-version).
+
+```bash
+curl -X GET 'http://127.0.0.1:7700/version'
+```
+
+The response should look something like this:
+
+```json
+{
+  "commitSha":"stringOfLettersAndNumbers",
+  "buildDate":"YYYY-MM-DDTimestamp",
+  "pkgVersion":"x.y.z"
+}
+```
+
+If you get the error `Cannot open database, expected MeiliSearch engine version: 0.X.X, current engine version 0.Y.Y`, your database is not compatible with the currently installed MeiliSearch version.
+
+In this case, you need to **download the compatible version now** (i.e. `0.X.X` in the above error message) so that you can access and export your database.
+
+:::: tabs
+
+::: tab cURL
+If you downloaded MeiliSearch using `curl`, find and download the compatible version [here](https://github.com/meilisearch/MeiliSearch/releases) before continuing.
+:::
+
+::: tab Homebrew
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+brew install meilisearch@0.X.X
+```
+
+:::
+
+::: tab Source
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+# clone MeiliSearch and checkout the branch of the expected version
+git clone https://github.com/meilisearch/MeiliSearch
+cd MeiliSearch
+git checkout v0.X.X
+
+# update the rust toolchain to the latest version
+rustup update
+
+# compile the project
+cargo build --release
+
+# execute the server binary
+./target/release/meilisearch
+```
+
+:::
+
+::: tab APT
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+apt-get install meilisearch=0.X.X
+```
+
+:::
+
+::: tab Docker
+
+Replace `0.X.X` with the version you would like to install.
+
+```bash
+docker run -it --rm \
+    -p 7700:7700 \
+    -v $(pwd)/data.ms:/data.ms \
```

Comment:
I missed it at first, but `$(pwd)/data.ms:/data.ms` needs to be within double-quotes so if `$(pwd)` contains a space, it won't return an error. I fixed it in the MeiliSearch repo, but we may have this elsewhere.

Reply:
Wow, nice catch!